### DISCUSSION TITLE
Image modifiers within the `grid:file` tag pair

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4574] [Feature Requests]
### Suggest an idea for ExpressionEngine

Please enable the use of image modifiers within the `file_grid:file` tag pair. 

This approach currently is not working:

```
{images_grid}
  {images_grid:file}
    {layout:set:append name="images_grid_1200px"}{url:1200px:webp}{/layout:set:append}
    {layout:set:append name="images_grid_600px"}{url:600px:webp}{/layout:set:append}
  {/images_grid:file}
{/images_grid}
```

**Comment by intoeetive** [comment]
I can confirm you cannot currently chain modifiers when using `{images_grid:file}{url:1200px:webp}{/images_grid:file}`

You can still however do this with a single variable like `{images_grid:file:1200px:webp}` outside of variable pair
